package.json

```diff
@@ -27,6 +27,7 @@
         "@babel/core": "^7.5.5",
         "@babel/preset-env": "^7.5.5",
         "babel-loader": "^8.0.6",
+        "babel-polyfill": "^6.26.0",
```

Comment:
https://babeljs.io/docs/en/6.26.3/babel-polyfill
> Because this is a polyfill (which will run before your source code), we need it to be a dependency, not a devDependency

Reply:
Right, was changed to the last version:)